vcx/libvcx/src/api/vcx.rs

```diff
@@ -28,7 +28,7 @@ use std::ffi::CString;
 pub extern fn vcx_init_with_config(command_handle: u32,
                                    config: *const c_char,
                                    cb: Option<extern fn(xcommand_handle: u32, err:u32)>) -> u32 {
-
+    ::utils::logger::LoggerUtils::init();
```

Comment:
This change will fix the logging on android but break it everywhere else (because the log_config hasn't been processed yet).

Reply:
ok, for now I will remove these init calls, till I figure out a clean way. There are few approaches that we can take, an if condition that checks for android platform, or a feature macro based approach or we can have match statements and we can call init on the basis of platforms. I will raise another PR for that